pkg/server/application_api/storage_inspection_test.go

```diff
@@ -336,50 +336,6 @@ func BenchmarkAdminAPIDataDistribution(b *testing.B) {
 	b.StopTimer()
 }
 
-func TestHotRangesResponse(t *testing.T) {
-	defer leaktest.AfterTest(t)()
-	defer log.Scope(t).Close(t)
-
-	defer serverutils.TestingSetDefaultTenantSelectionOverride(
-		// bug: HotRanges not available with secondary tenants yet.
-		base.TestIsForStuffThatShouldWorkWithSecondaryTenantsButDoesntYet(109499),
-	)()
-
-	srv := rangetestutils.StartServer(t)
-	defer srv.Stopper().Stop(context.Background())
-	ts := srv.ApplicationLayer()
-
-	var hotRangesResp serverpb.HotRangesResponseV2
-	if err := srvtestutils.PostStatusJSONProto(ts, "v2/hotranges", &serverpb.HotRangesRequest{}, &hotRangesResp); err != nil {
```

Comment:
Note that we're using the active endpoint for this test. I'd keep this unless there's a reason we wouldn't?